### PR TITLE
Adds a lightweight section31 operative as seen in 2cw's PR

### PR DESCRIPTION
**Pull request by Kmc2000**
:cl: 2cwldys
add: May or may not have added a section31 antag, which does not exist
/:cl:

This is a rework of #66 but more light-weight and with a few code fixes.

### CODE REVIEW

**Comment by 2cwldys** [comment]
Glad to see it has some use!

File: DS13/code/modules/antagonists/section31.dm [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+	roundend_category = "Section 31 Agents"
+	antagpanel_category = "Section 31 Agent"
+	job_rank = ROLE_SECTION31
+	antag_moodlet = /datum/mood_event/focused
+	var/special_role = ROLE_SECTION31
+	var/section31_outfit = /datum/outfit/section31
+	can_hijack = HIJACK_HIJACKER
+
+/datum/antagonist/section31/on_gain()
+	SSticker.mode.traitors += owner
```
Comment (alexkar598) [inline DS13/code/modules/antagonists/section31.dm]: What if the mode is clings or extended

File: DS13/code/modules/antagonists/section31.dm [hunk 2]
```diff
@@ -0,0 +1,92 @@
+/datum/antagonist/section31
+	name = "Section 31 Agent"
+	roundend_category = "Section 31 Agents"
+	antagpanel_category = "Section 31 Agent"
+	job_rank = ROLE_SECTION31
+	antag_moodlet = /datum/mood_event/focused
+	var/special_role = ROLE_SECTION31
+	var/section31_outfit = /datum/outfit/section31
```
Comment (alexkar598) [inline DS13/code/modules/antagonists/section31.dm]: Useless var that removes to code readability,inline it

File: DS13/code/modules/antagonists/section31.dm [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+	var/mob/living/carbon/human/H = owner.current
+
+	H.set_species(/datum/species/human) // All Section 31 members seen so far are human.
+	H.grant_language(/datum/language/codespeak)
+	H.equipOutfit(section31_outfit)
+	give_codewords()
+	return TRUE
+
+/datum/antagonist/section31/on_removal()
+	//Remove malf powers.
```
Comment (alexkar598) [inline DS13/code/modules/antagonists/section31.dm]: Um what?

File: DS13/code/modules/antagonists/section31.dm [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+	SSticker.mode.traitors -= owner
```
Comment (alexkar598) [inline DS13/code/modules/antagonists/section31.dm]: See comment in on_gain

File: DS13/code/modules/antagonists/section31.dm [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+	if(!silent && owner.current)
+		to_chat(owner.current,"<span class='userdanger'>You have been deactivated, and are no longer a [special_role]!</span>")
+	owner.special_role = null
```
Comment (alexkar598) [inline DS13/code/modules/antagonists/section31.dm:29]: Revoke code speak

File: DS13/code/modules/antagonists/section31.dm [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+	to_chat(owner.current, "<font size=2 color=cyan>https://en.wikipedia.org/wiki/Section_31</font>")
+	to_chat(owner.current, "<font size=2 color=cyan>You are here under best interests of the higher echelons of Starfleet Command, we operate under what is known as Section 31.</font>")
+	to_chat(owner.current, "<font size=2 color=cyan>You are allowed to kill, torture and maim in the interests of your objectives. But remember that the crew of the [station_name()] are your allies. You are equipped with several nonlethal options for dealing with your comrades.</font>")
+	to_chat(owner.current, "<font size=2 color=cyan>No one of any authority is permitted to interfere with your objectives, complete them at all costs. Do not allow yourself to be captured. To this effect you are armed with a cyanide pill and an explosive implant. Remove all nuisances and potential threats to yourself and the Federation by any means necessary.</font>")
+	var/datum/objective/O = new /datum/objective/survive() //Change this later
+	O.owner = owner
+	objectives += O
+	owner.announce_objectives()
+	give_codewords()
+
```
Comment (alexkar598) [inline DS13/code/modules/antagonists/section31.dm:45]: Code words are already given in on_gain

File: DS13/code/modules/antagonists/section31.dm [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+	name = "Section 31 Agent"
+	uniform = /obj/item/clothing/under/trek/command/ds9
+	accessory = /obj/item/clothing/accessory/ds9_jacket
+	id = /obj/item/card/id/syndicate
+	belt = /obj/item/gun/energy/phaser
+	l_pocket = /obj/item/pda
+	r_pocket = /obj/item/melee/classic_baton/telescopic
+	ears = /obj/item/radio/headset
+	shoes = /obj/item/clothing/shoes/jackboots
+	back = /obj/item/storage/backpack/duffelbag/syndie
```
Comment (alexkar598) [inline DS13/code/modules/antagonists/section31.dm]: Syndicate duffel bag? Come on,is it ss13 now?

**Review by alexkar598 — CHANGES_REQUESTED** [review]

File: DS13/code/modules/antagonists/section31.dm [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+	new /obj/item/reagent_containers/pill/cyanide(src)
+	new /obj/item/reagent_containers/glass/bottle/chloralhydrate(src)
+	new /obj/item/reagent_containers/hypospray(src)
+	new /obj/item/reagent_containers/syringe/mulligan(src)
+	new /obj/item/reagent_containers/syringe/mulligan(src)
+
+/obj/item/storage/box/section31/combat/PopulateContents()
+	new /obj/item/card/emag(src)
+	new /obj/item/kitchen/knife/combat(src)
+	new /obj/item/chameleon(src)
```
Comment (alexkar598) [inline DS13/code/modules/antagonists/section31.dm:90]: W-w-what?

**Review by alexkar598 — APPROVED** [review]
all abord the merge train

**Pull request merged**